python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -159,7 +157,6 @@ def _update(self):
             self.provider.internal_ip(node_id)
             for node_id in self.all_workers()
         ])
-        self.log_info_string(nodes)
```

Comment:
Why remove that? this is printed before anything changes and it seems to me like a good thing to have.

Reply:
To only print one info string per update loop.

Reply:
Can you make the middle ones debug and the external one info?
From my personal experience, without the intermediate ones we will have a have time figuring out the bugs. e.g.,
request_resources() terminated in the beginning and launched nodes at the end, with this log we will never observe that the node was removed and readded.

Reply:
this may actually be problematic because the summary requires a describe instances call

Reply:
Is there any way to still be able to debug this?

Reply:
i think the best thing to do would be to add more specific debug logs in those places, or solve the more general problem by improving our caching layer

Reply:
Ok since we're actively having issues with API calls right now, I'm going to remove these logs for now. We can add them, or something similar back in the future.